docs/build_and_release.md

```diff
@@ -0,0 +1,90 @@
+# Build
+The `acs-packaging` project uses _Travis CI_. \
+The `.travis.yml` config file can be found in the root of the repository.
+
+
+## Stages
+Although a little unusual, builds of branches other than `master` and `release` branches are preceded by
+cloning and building branches with the same name in upstream projects: `alfresco-community-repo` and
+`alfresco-enterprise-repo`. This is done so that development can take place in parallel between projects
+without having to do development releases to link them together. In fact, you don't even have to wait for
+the upstream Travis build to complete. You just need to make sure the parent poms reference
+each other and the `dependency.alfresco-community-repo.version` and
+`dependency.alfresco-enterprise-repo.version` use the same SNAPSHOT values.
+
+1. **test**: Java build with unit tests, integration tests and WhiteSource scan.
+2. **docker_latest**: 
+3. **release**: 
+3. **publish**: Artifact deployment to AWS Release bucket.
+
+
+ALL BELOW IS COPIED FROM ANOTHER PROJECT
+
```

Comment:
This .md file (mainly from this point) was copied from alfresco-transform-core project. As a result, the next commit to master after this PR was merged was to tidy it up so this was not included.